xml/http_servers.xml

```diff
@@ -2,6 +2,14 @@
 <fingerprints matches="http_header.server" protocol="http" database_type="service" preference="0.90">
   <!-- HTTP Server headers are matched against these patterns to fingerprint HTTP servers. -->
 
+  <fingerprint pattern="^Transmission$">
```

Comment:
This doesn't match up with what I'm seeing. What version are you fingerprinting?

```
$ curl -s -u admin:admin http://localhost:9091/transmission/web/ | grep -i "<title"
    <title>Transmission Web Interface</title>
```

Reply:
I'm not sure what I ended up doing here, but I think I probably shouldn't have been reviewing the PR when I was tired! Thank you for rolling along with my messy review. 😅